[2021-07-08 13:26:40,260] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:26:37.545219+00:00 [queued]>
[2021-07-08 13:26:40,302] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:26:37.545219+00:00 [queued]>
[2021-07-08 13:26:40,302] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:26:40,302] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 13:26:40,302] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 13:26:40,313] {taskinstance.py:1107} INFO - Executing <Task(PostgresOperator): query_users_data> on 2021-07-08T13:26:37.545219+00:00
[2021-07-08 13:26:40,324] {standard_task_runner.py:52} INFO - Started process 18755 to run task
[2021-07-08 13:26:40,333] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'query_users_data', '2021-07-08T13:26:37.545219+00:00', '--job-id', '143', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmp8rkegp1c', '--error-file', '/tmp/tmpi68f0fen']
[2021-07-08 13:26:40,343] {standard_task_runner.py:77} INFO - Job 143: Subtask query_users_data
[2021-07-08 13:26:40,413] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.query_users_data 2021-07-08T13:26:37.545219+00:00 [running]> on host 3cf0296804bd
[2021-07-08 13:26:40,469] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=query_users_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T13:26:37.545219+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T13:26:37.545219+00:00
[2021-07-08 13:26:40,471] {postgres.py:68} INFO - Executing: 
            SELECT * FROM users;
        
[2021-07-08 13:26:40,480] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 13:26:41,607] {dbapi.py:204} INFO - Running statement: 
            SELECT * FROM users;
        , parameters: None
[2021-07-08 13:26:41,980] {dbapi.py:212} INFO - Rows affected: 2
[2021-07-08 13:26:42,178] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=query_users_data, execution_date=20210708T132637, start_date=20210708T132640, end_date=20210708T132642
[2021-07-08 13:26:42,215] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-08 13:26:42,232] {local_task_job.py:151} INFO - Task exited with return code 0
